content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,256 @@
+  :guilabel:`Safety Stock Target`.
+- :guilabel:`Yellow` means that a replenishment order has already been generated, but the quantity
+  it was created for is not enough to keep stock at the :guilabel:`Safety Stock Target`.
+- :guilabel:`Red` means that a replenishment order has already been generated, but the quantity it
+  was created for puts the amount of stock above the :guilabel:`Safety Stock Target`.
+
+The :guilabel:`+ Suggested Replenishment` field appears white if no replenishment order has been
+generated, and it is not necessary to generate one at the current moment.
+
+Add a product
+=============
 
-.. tip::
-        In the MPS settings, you can define the time range of your MPS (month/week/day)
-        and the number of periods you want to display at all times.
+To use |MPS| to manage the replenishment of a product, navigate to :menuselection:`Manufacturing app
+--> Planning --> Master Production Schedule`. At the top of the |MPS| page, click :guilabel:`Add a
+Product` to open the :guilabel:`Add a Product` pop-up window.
 
-Now, go to :menuselection:`Planning --> Master Production Schedule` and click on *add a
-product*. You can now define your safety stock target (= the stock you
-want to have on hand at the end of the period) and the minimum and
-maximum quantities that must or can be replenished in each period.
+.. important::
+   Products must be properly configured to be replenished through the |MPS|.
 
-.. image:: use_mps/mps_1.png
-    :align: center
+   In the case of manufactured products, the *Manufacture* route must be selected in the *Routes*
+   section of the *Inventory* tab, on the product's page.
 
-In the MPS view, you can decide which information you would like to display
-by clicking on *rows*. For instance, the *Actual demand* will show you which
-quantity of products has already been ordered for the period, or *Available to Promise*,
-what can still be sold during that same period (what you plan to replenish - what
-is already sold during the period). You can also decide to hide rows if you like.
+   In the case of products that are purchased, the *Buy* route must be selected in the *Routes*
+   section of the *Inventory* tab, on the product's page. Additionally, a vendor and the price they
+   sell the product for must also be specified on the *Purchase* tab.
 
-.. image:: use_mps/mps_2.png
-    :align: center
+On the pop-up window, select the product to add in the :guilabel:`Product` drop-down menu. If the
+product is replenished through manufacturing, select the product's |BoM| in the :guilabel:`Bill of
+Materials` field.
 
-Estimate your demand and launch replenishment
-=============================================
+.. note::
+   Selecting a BoM when adding a product to the |MPS| also adds any components listed on the BoM. If
+   it is not necessary to manage the replenishment of components through the |MPS|, simply leave the
+   :guilabel:`Bill of Materials` field blank.
 
```

Comment:
I know you mentioned that this field didn't pop up for you during your revisions, and I fully saw that -- however, during my 2nd round of revisions, I came across it again. So, I think this might be worth double-checking/looking into... 
![Screenshot 2024-06-26 061658](https://github.com/odoo/documentation/assets/73958186/2e5e9126-df46-4183-bdde-2b394ba3f9da)